src/main/java/de/uhd/ifi/se/decision/management/jira/rest/ConfigRest.java

```diff
@@ -715,22 +715,103 @@ public Response classifyWholeProject(@Context HttpServletRequest request,
 	/* Configuration for Consistency		*/
 	/*										*/
 	/* **************************************/
-	@Path("/setConsistencyActivated")
+	@Path("/setMinimumLinkSuggestionProbability")
 	@POST
-	public Response setConsistencyActivated(@Context HttpServletRequest request,
-											@QueryParam("projectKey") String projectKey,
-											@QueryParam("isConsistencyActivated") String isActivatedString) {
-		Response response = this.checkRequest(request, projectKey, isActivatedString);
-		if (response != null) {
+	public Response setMinimumLinkSuggestionProbability(@Context HttpServletRequest request,
+														@QueryParam("projectKey") String projectKey,
+														@QueryParam("minLinkSuggestionProbability") double minLinkSuggestionProbability) {
+		Response response = this.checkIfDataIsValid(request, projectKey);
+		if (response.getStatus() != 200) {
+			return response;
+		}
+		response = checkIfProjectKeyIsValid(projectKey);
+		if (response.getStatus() != Status.OK.getStatusCode()) {
+			return response;
+		}
+		if (!(1. >= minLinkSuggestionProbability && minLinkSuggestionProbability >= 0.)) {
+			return Response.status(Status.BAD_REQUEST).entity(ImmutableMap.of("error", "The minimum of the score value is invalid."))
+				.build();
+		}
+
+		ConfigPersistenceManager.setMinLinkSuggestionScore(projectKey, minLinkSuggestionProbability);
+		return Response.ok(Status.ACCEPTED).build();
+	}
+
+	@Path("/setMinimumDuplicateLength")
+	@POST
+	public Response setMinimumDuplicateLength(@Context HttpServletRequest request,
+											  @QueryParam("projectKey") String projectKey,
+											  @QueryParam("minDuplicateLength") int minDuplicateLength) {
+		Response response = this.checkIfDataIsValid(request, projectKey);
+		if (response.getStatus() != 200) {
 			return response;
 		}
 		response = checkIfProjectKeyIsValid(projectKey);
 		if (response.getStatus() != Status.OK.getStatusCode()) {
 			return response;
 		}
-		boolean isActivated = Boolean.parseBoolean(isActivatedString);
-		ConfigPersistenceManager.setConsistencyActivated(projectKey, isActivated);
+		if (!(minDuplicateLength >= 3)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use opposite operator instead of the logic complement operator.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5586048)